src/main/scala/tech/cryptonomic/conseil/tezos/TezosRemotes.scala

```diff
@@ -0,0 +1,202 @@
+package tech.cryptonomic.conseil.tezos
+
+import tech.cryptonomic.conseil.config.{HttpStreamingConfiguration, NetworkCallsConfiguration}
+import tech.cryptonomic.conseil.config.Platforms.TezosConfiguration
+import tech.cryptonomic.conseil.generic.chain.RemoteRpc
+import tech.cryptonomic.conseil.util.JsonUtil.JsonString
+
+
+object TezosRemoteInstances {
+
+  object Akka {
+    import com.typesafe.scalalogging.Logger
+    import cats.Monoid
+    import scala.concurrent.Future
+    import akka.actor.ActorSystem
+    import akka.stream.ActorMaterializer
+    import akka.stream.scaladsl.Source
+    import akka.http.scaladsl.Http
+    import akka.http.scaladsl.model._
+
+    case class RemoteContext(
```

Comment:
Maybe the entire file/object should be TezosConfiguration/TezosContext? I think a rename is in order.

Reply:
🤔 ... ok, what about TezosRemoteContext, or TezosRpcContext ? It has to do with how to connect to the node to make a REST call.
Still we need to think of something that will make sense in perspective, also for Eth and other platforms